src/db/dbQuery.ts

```diff
@@ -0,0 +1,20 @@
+import { database } from "./dbConfig";
+
+
+export const getAllUnsentEmailData =() => {
+    try {
+        return new Promise((resolve, reject) => {
+            database.query('select * from notification where email=true and email_sent=false and seen=false;', function (err, result){
```

Comment:
That is the first step.
 You still need to:
> 2. build the email body based on the content of the event,
> 3. send the email to the right users (they have to have subscribed to email notification), and
> 4. set in the database that the emails were sent

Reply:
Done